src/main/java/com/autotune/analyzer/recommendations/engine/DurationBasedRecommendationEngine.java

```diff
@@ -516,12 +516,12 @@ private boolean populateRecommendation(String recommendationTerm,
         if (numPods == 0) {
             RecommendationNotification recommendationNotification = new RecommendationNotification(RecommendationConstants.RecommendationNotification.ERROR_NUM_PODS_CANNOT_BE_ZERO);
             notifications.add(recommendationNotification);
-            LOGGER.error("Number of pods cannot be zero");
+            LOGGER.debug("Number of pods cannot be zero");
             isSuccess = false;
         } else if (numPods < 0) {
             RecommendationNotification recommendationNotification = new RecommendationNotification(RecommendationConstants.RecommendationNotification.ERROR_NUM_PODS_CANNOT_BE_NEGATIVE);
             notifications.add(recommendationNotification);
-            LOGGER.error("Number of pods cannot be negative");
+            LOGGER.debug("Number of pods cannot be negative");
```

Comment:
Please revert all the LOGGER changes as they appear to be for debugging

Reply:
done